### DISCUSSION TITLE
https://blog.3place.cn/3place/fsd/FSD030.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#33] [General]
# https://blog.3place.cn/3place/fsd/FSD030.html

https://blog.3place.cn/3place/fsd/FSD030.html

**Comment by picklecai** [comment]
你的老师是贤达之士：
谁不认识高贵的瓦格纳博士，
他是当今学界中的泰斗！
他是团结学界的唯一的名流，
他使学术日益丰富。
求知欲强的好学之徒
纷纷聚到他的身旁。
他独霸讲坛，光芒盖世，
像圣彼得掌管着钥匙，
打开上界下界的知识之窗。

真不敢相信，这是瓦格纳。

**Comment by picklecai** [comment]
最孱弱的博学之士，
弄成烧炭夫的样子，
从耳到鼻，都像黑炭，
由于吹火，变成红眼，
他时时刻刻，求成心切，
把火钳之声当作奏乐。

瓦格纳，不是疯了就是傻了。

**Comment by picklecai** [comment]
就是学者也要继续钻，
他舍此无所作为。
人们就这样用纸牌搭座房子，
可是最大的天才也不能竣事。

house of cards的引申义：这砌成的房子，外观可能看起来很雄伟，但是却危如累卵，一击即破。暗指主角游走在政治独木桥上，需要不断平衡各方力量，否则整个布局就会瞬间全塌。

浮士德出去逛了一圈，现实中并无任何改变。仍然是生命浪费在无意义的事情上。瓦格纳看不透，孜孜以求，白白傻了一个人。

**Comment by picklecai** [comment]
当年我并未将你看轻；
看到幼虫和蛹，就可以预知
彩色的蝴蝶将会成形。

梅菲斯特一时兴起，逗了那个学生一次。没想到他以后也成为了学者。

**Comment by picklecai** [comment]
经验！就像泡沫和尘屑！
它跟精神难以匹敌。
承认吧！从来所知的一切，
根本没有一知的价值……

这孩子也是个虚无主义者？还是继续上文看法，瓦格纳教出来的，无非老调重弹，所以经验毫无意义？

**Comment by picklecai** [comment]
人的生命存在于血液之中，
哪有像青年的血那样生动？
从生命之中创造新的生命，
就靠生动的血液鼓足干劲。
那里一切在活动，有所成功，
弱者倒下，有为者奋勇前冲。

想当年，意气风发，指点江山，激扬文字，觉得整个世界都在脚下。

**Comment by picklecai** [comment]
老年是一种怕冷的热病，
寒飕飕地苦于奇想，
一个人过了三十岁年龄，
他就已经像死了一样。

单看第一句，就觉得很神奇。什么是热病？什么是怕冷？从第二句明白了，抠抠搜搜是怕冷，想半天（苦于奇想））是热，但又不干还是怕冷。

**Comment by picklecai** [comment]
本没有世界，是我把它造成；
我把太阳从大海里面捞起；
月亮的盈亏完全自我而始；
白昼在我的路上炫耀光华，
大地迎迓我以万绿和百花。
在那最初的一夜，一切繁星
随着我的眼色而大放光明。
除了我，还有谁能将你们解放，
摆脱世俗狭隘的思想框框？

浮士德不在的日子里，瓦格纳傻了，学生疯了。

**Comment by picklecai** [comment]
我们现在不必为他担忧：
葡萄汁尽管发酵得不成样子，
最后总会成为美酒。

梅菲斯特是个好爸爸
